src/App.jsx

```diff
@@ -37,14 +37,15 @@ export default function App() {
       <Toaster toastOptions={toastOptions} position="top-right" />
       <Header icons={icons} setIcons={setIcons} />
       <div className={styles.Content}>
-        {!!icons.length && (
+        {!!icons.length ? (
           <IconPreviewArea
             iconSet={iconSet}
             icons={icons}
             setIcons={setIcons}
           />
+        ) : (
+          <LandingSection />
         )}
```

Comment:
Maybe it could be better like this.

```jsx
{icons.length ? (
    <LandingSection />
  ) : (
    <IconPreviewArea
      iconSet={iconSet}
      icons={icons}
      setIcons={setIcons}
    />
)}
```